docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1489,6 +1489,13 @@ How to **fix** it:
 - Ensure each compactor replica has successfully updated bucket index of each owned tenant within the double of `-compactor.cleanup-interval` (query below assumes the cleanup interval is set to 15 minutes):
   `time() - cortex_compactor_block_cleanup_last_successful_run_timestamp_seconds > 2 * (15 * 60)`
 
+### err-mimir-histogram-different-number-spans-buckets
+
+This non-critical error occurs when Mimir receives a write request that contains a native (also known as sparse) histogram whose either negative or positive spans specify
+a different number of buckets then there are provided.
+
+> **Note**: Series with invalid histograms are skipped during the ingestion, and series within the same request are ingested.
```

Comment:
I did not understand the `series within the same request are ingested` part

Reply:
I took it from one of the other errors, but I presume it means: "other series that don't have invalid samples/histograms that are part of the same request are still ingested"